examples/stm32/xively_demo_stm32f4xx_nucleo_wifi/Src/demo_io.c

```diff
@@ -0,0 +1,358 @@
+#include <assert.h>
```

Comment:
ok, downloaded the mems SDK and haven't found this file, so you put this file together?